### ISSUE TITLE
contrib is a file

### ISSUE THREAD

**Issue by jeremyeder** [distributed-system-analysis/pbench#288]
@ndokos contrib is a "file", rather than a "directory" since #204 was merged.

```
# rpm -q pbench-agent
pbench-agent-0.39-1gaf1ffe4.noarch
# file /opt/pbench-agent/bench-scripts/contrib 
/opt/pbench-agent/bench-scripts/contrib: Python script, ASCII text executable
```

Was the plan to create a directory bench-scripts/contrib and then move pbench_bzt inside it?

**Comment by ndokos** [comment]
It was - not sure what happened, but I'll fix.

**Comment by ndokos** [comment]
This was a packaging error in the spec file: I now create the contrib/ directory and the pbench-bzt file gets copied properly. This should be fixed in 0.39-2 and later.
